src/test/scala/org/ergoplatform/nodeView/state/ErgoStateSpecification.scala

```diff
@@ -21,9 +21,10 @@ class ErgoStateSpecification extends ErgoPropertyTest {
       val toRemove = changes.toRemove.map(_.boxId)
       toRemove.count(boxId => java.util.Arrays.equals(toRemove.head, boxId)) shouldBe 2
 
-      val dsRoot = BlockTransactions.transactionsRoot(dsTxs)
+      val dsRoot = BlockTransactions.transactionsRoot(dsTxs, Header.InitialVersion)
```

Comment:
I think this property should be executed once for each supported version.

Reply:
done